src/utils/maak_protocol.R

```diff
@@ -0,0 +1,263 @@
+#' @title Create a folder with a markdown template to start working on a new fieldwork protocol
+#'
+#' @description This function will create a new folder based on values that are passed on via the parameters and creates a R-markdown skeleton based on a template file to start working on a new protocol.
+#'
+#' @param titel
+#' @param subtitel
+#' @param auteurs
+#' @param datum
+#' @param reviewers
+#' @param documentbeheerder
+#' @param revisie
+#' @param projectnaam
+#' @param thematiek
+#'
+#' @return A new subfolder beneath src with an Rmarkdown file
+#'
+#' @importFrom rprojroot find_root
+#' @importFrom rprojroot is_git_root
+#' @importFrom whisker whisker.render
+#' @importFrom purrr map_chr
+#' @importFrom stringr str_replace_all
+#' @importFrom assertthat
+#' assert_that
+#' is.string
+#' is.date
+#' @examples
+#' maak_protocol()
+#'
+maak_svp <- function(
+  titel = "Titel",
+  subtitel = "",
+  korte_titel = "korte titel",
+  auteurs = "voornaam1 naam1, voornaam2 naam2, ...",
+  datum = Sys.Date(),
+  reviewers = "voornaam1 naam1, voornaam2 naam2, ...",
+  documentbeheerder = "voornaam naam",
+  revisie = "0.1",
+  projectnaam,
+  thematiek = c("generiek", "water", "lucht", "bodem", "vegetatie", "soorten")) {
+  
+  # needed for as long as this function is not part of a package
+  require(assertthat)
+  require(rprojroot)
+  require(whisker)
+  require(purrr)
+  require(stringr)
+ 
+  
+  # check parameters
+  assert_that(is.string(titel))
+  assert_that(is.string(subtitel))
+  assert_that(is.string(korte_titel) & length(korte_titel) <= 20)
```

Comment:
```suggestion
  assert_that(is.string(korte_titel), nchar(korte_titel) <= 20)
```
`length()` counts elements in the vector. `nchar()` counts characters